### DISCUSSION TITLE
Sam detects 53k free games...?

### DISCUSSION THREAD

**Discussion by zImMateja** [gibbed/SteamAchievementManager#415] [Q&A]
![image_2024-09-16_050917186](https://github.com/user-attachments/assets/f17b0f53-20e2-4b17-85cf-b84767133165)

**Comment by gibbed** [comment]
That would be every game SAM checks for, so something is wrong with your Steam client if it's saying you own all of them (unless you actually own all of them).

**Comment by gibbed** [comment]
Have you changed anything in your Steam settings that is out of the ordinary? Are you using Family Share, etc?

**Comment by gibbed** [comment]
#416
